tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+            )
+
+        return integration_models[model_type]
+
+    def get_model_alternatives(self, integration: str) -> list:
+        """Get alternative models for a integration"""
+        if integration not in self._config["models"]:
+            raise ValueError(f"Unknown integration: {integration}")
+
+        return self._config["models"][integration].get("alternatives", [])
+
+    def get_model_capabilities(self, model: str) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)